src/robobo/simulation.py

```diff
@@ -28,6 +34,10 @@ def connect(self, address='127.0.0.1', port=19999):
         while time.time() - startTime < get_handles_timeout:
             try:
                 self.initialize_handles()
+                print(f"setting realtime={realtime}")
+                vrep.simxSetBooleanParameter(
```

Comment:
could you wrap this function in `vrep.unwrap_vrep` ? so that if it returns an error and fails to set real time, students will know?